articles/wikis.md

```diff
@@ -88,6 +88,7 @@ See the 'Insert' list for the following widgets to customize your Wiki design.
 | -- | :--: |
 | **References** <br/> Create a reference list by linking to papers using the References widget | <img id="image" src="../assets/images/wiki_reference.png"> |
 | **File Preview** <br/> Embeds a preview window for csv, txt and image files||
+| **Image** <br/> Embeds an JPG or PNG uploaded, from the web, or from a synapse file. **Note:** Images with the word 'ad' in it will be blocked if you are using an adblocker. ||
```

Comment:
```suggestion
| **Image** <br/> Embeds an JPG or PNG. The image can be uploaded, from the web, or from a Synapse file. **Note:** Images with the word 'ad' in it will be blocked if you are using an adblocker. ||
```

Reply:
Are you sure that we only show JPG and PNG? What about GIF?